### PR TITLE
Team event type scroll, manage team invite from mobile

### PR DESCRIPTION
**Pull request by leog**
## What does this PR do?

Fixes a couple of UI bugs.

#### Original Mobile:
image

#### Original Desktop:
Teams_accept_buttons

#### Fixed Mobile:
Teams_accept_with_dropdown
Screenshot 2022-06-27 at 12 39 17

#### Original Mobile
image

#### Fixed Mobile
https://user-images.githubusercontent.com/467258/176039791-f7fe4724-5045-4e32-967e-d726b6df95e5.mov

**Environment**: Staging(main branch) / Production

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

**Review by pumfleet — APPROVED** [review]
Code looks good to me

**Comment by Jaibles** [comment]
Looks great 👍

**Review by PeerRich — APPROVED** [review]

**Review by zomars — APPROVED** [review]

File: apps/web/components/eventtype/CreateEventType.tsx [hunk 1]
```diff
@@ -200,16 +200,43 @@ export default function CreateEventTypeButton(props: Props) {
...
+                <TextField
+                  label={t("url")}
+                  required
+                  addOnLeading={
+                    <InputLeading>
+                      {process.env.NEXT_PUBLIC_WEBSITE_URL}/{pageSlug}/
+                    </InputLeading>
+                  }
+                  {...register("slug")}
+                />
```
Comment (zomars) [inline apps/web/components/eventtype/CreateEventType.tsx:237]: Can we prevent registering multiple fields? (both desktop and mobile) Also I think we can apply the visual fixes more granularly.

**Review by zomars — CHANGES_REQUESTED** [review]
May be related with registering multiple fields (even if only one is visible) but this test is failing:
```
[chromium] › ../../apps/web/playwright/event-types.test.ts:34:5 › Event Types tests › pro user › can add new event type 
[918](https://github.com/calcom/cal.com/runs/7093531075?check_suite_focus=true#step:9:919)
```

**Comment by baileypumfleet** [comment]
@leog any update on this?

**Comment by leog** [comment]
> @leog any update on this?



> @leog any update on this?

Will be merged today hopefully, returning from sick leave today after 2 days.

File: apps/web/components/eventtype/CreateEventType.tsx (hunk 1 above)
Comment (leog) [reply apps/web/components/eventtype/CreateEventType.tsx]: Simplified, no special responsive treatment.

**Review by zomars — APPROVED** [review]

**Pull request merged**